docker/buoy/Dockerfile

```diff
@@ -8,8 +8,14 @@ RUN apt update \
         wget
 
 # Using non-official Gazebo + ROS combination, set it explicitly
-ENV IGNITION_VERSION fortress
-ENV GZ_VERSION fortress
+ENV GZ_VERSION garden
```

Comment:
I wonder if we should add cyclone here too... although it doesn't affect local tests. We should settle on one configuration and everybody uses the same thing.
```
# for cyclonedds rmw implementation
apt install -y ros-humble-rmw-cyclonedds-cpp

export RMW_IMPLEMENTATION=rmw_cyclonedds_cpp
```

Reply:
oops, missed this one in the dockerfile!